packages/mdc-typography/_mixins.scss

```diff
@@ -40,3 +40,28 @@
   white-space: nowrap;
   overflow: hidden;
 }
+
+@mixin mdc-typography-baseline-top($distance) {
+  margin-top: 0;
+  line-height: normal;
```

Comment:
I think setting line-height to normal will become unnecessary once we remove line-height from typography scale.

But for now I think it is okay.

Reply:
I would keep this for now. Secondary text in list and hint text on text field still has line-heights.